cli/ultra.py

```diff
@@ -69,10 +69,11 @@ def build():
 @click.argument("auth_key", type=str, default=None)
 @click.argument("port", default=7432, type=int)
 def worker(auth_key, port):
-    from smarts.zoo.worker import listen
+    from smarts.zoo.worker import serve
 
     auth_key = auth_key if auth_key else ""
-    listen(port, auth_key)
+    # listen() does not exist anymore, does serve() provide the same function?
+    serve(port)
```

Comment:
@Adaickalavan 

Just be careful here since I believe ULTRA is using a different version of SMARTS than the current version. We may want to provide the following:

```suggestion
    auth_key = auth_key if auth_key else ""
    try:
        from smarts.zoo.worker import listen
        listen(port, auth_key)
    except ImportError:
        from smarts.zoo.worker import serve
        serve(port)
```

Reply:
The latest version of `ultra` code was developed separately in the `ultra-develop` branch. 

This cli command `scl ultra worker` is for manually starting a manager in a distributed computing setting. This version of cli command code appears to use the original implementation from smarts version `v0.4.10` : https://github.com/huawei-noah/SMARTS/blob/v0.4.10/smarts/zoo/worker.py .

I believe the correct cli command is identical to `scl zoo manager`, thus we might simply delete this entire `scl ultra worker` cli command.